cmd/dispatcher/main.go

```diff
@@ -41,14 +41,15 @@ type envConfig struct {
 	// Should failed deliveries be requeued in the RabbitMQ?
 	Requeue bool `envconfig:"REQUEUE" default:"false" required:"true"`
 
+	// Number of concurrent messages in flight
+	PrefetchCount int           `envconfig:"PREFETCH_COUNT" default:"10" required:"false"`
 	Retry         int           `envconfig:"RETRY" required:"false"`
 	BackoffPolicy string        `envconfig:"BACKOFF_POLICY" required:"false"`
 	BackoffDelay  time.Duration `envconfig:"BACKOFF_DELAY" required:"false"`
 }
 
 const (
 	defaultBackoffDelay = 50 * time.Millisecond
```

Comment:
I would set this as a `BackoffDelay` default.

Reply:
Not sure what you mean. The prefetch count in milliseconds should be the backoff delay?